lib/tsc-watch.js

```diff
@@ -29,46 +192,6 @@ const {
   args,
 } = extractArgs(process.argv);
 
-function killProcesses(killAll) {
```

Comment:
I wasn't sure of the purpose of `killAll` and treating `firstSuccessKiller` differently from the others, so I haven't replicated that in my version. If this behavior needs to remain, let me know and I can put it in.